.github/workflows/release.yml

```diff
@@ -86,17 +36,23 @@ jobs:
       - uses: ./.github/actions/sccache
         with:
           key: ${{ env.TARGET }}
-      - run: cargo build -p cargo-risczero --release
-      - run: |
+      - if: matrix.os == 'Linux'
+        run: docker build -f .github/containers/release/Dockerfile --output tmp/pkg .
+      - if: matrix.os == 'macOS'
+        run: |
+          cargo build -p cargo-risczero --release
           mkdir -p tmp/pkg
           cp target/release/cargo-risczero tmp/pkg
           cp target/release/r0vm tmp/pkg
-          cd tmp/pkg && tar cv * | gzip -9 > ../cargo-risczero-$TARGET.tgz
+      - run: tar cv * | gzip -9 > ../cargo-risczero-$TARGET.tgz
+        working-directory: tmp/pkg
       - uses: svenstaro/upload-release-action@v2
         with:
+          overwrite: true
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           file: tmp/cargo-risczero-${{ env.TARGET }}.tgz
-          # tag: ${{ env.TAG }} # only for testing
+          tag: ${{ env.TAG }} # only for testing
+      - run: sleep 90
```

Comment:
?

Reply:
This is needed because otherwise the test that follows tries to download too soon and causes intermittent 404s. I don't know if `90` is long enough but seems to be more stable this way.

Reply:
That make sense, might be worth a comment inline to that effect.